Application.php

```diff
@@ -49,4 +64,18 @@ public function add(Command $command): ?Command
 
         return $return;
     }
+
+    protected function getDefaultInputDefinition(): InputDefinition
+    {
+        $inputDefinition = parent::getDefaultInputDefinition();
+        $inputDefinition->addOption(
+            new InputOption('--all-commands', null, InputOption::VALUE_NONE, 'Do not exclude commands')
```

Comment:
This will add a documentation about `--all-commands` in `list` command (the default one when calling `bin/console`.)